railties/lib/rails/generators/rails/app/templates/config/puma.rb.tt

```diff
@@ -13,7 +13,7 @@ threads min_threads_count, max_threads_count
 
 # Specifies that the worker count should equal the number of processors in production.
 if ENV["RAILS_ENV"] == "production"
-  worker_count = Integer(ENV.fetch("WEB_CONCURRENCY") { Concurrent.physical_processor_count })
+  worker_count = Integer(ENV.fetch("WEB_CONCURRENCY") { Concurrent.physical_processor_count }).to_i
```

Comment:
Does the `Integer()` wrapper not cover this?

Reply:
Yes. It does. I'm reverting